sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -885,6 +888,28 @@ public EventDirectorySteps(
               getByEventUuid(CreateNewEventSteps.newEvent.getUuid()));
         });
 
+    When(
+        "I search for the last event uuid created by UI",
+        () -> {
+          webDriverHelpers.fillInWebElement(
+              SEARCH_EVENT_BY_FREE_TEXT, CreateNewEventSteps.eventUUID);
+          webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTER);
```

Comment:
waitUntilANumberOfElementsAreVisibleAndClickable this can be added after apply, so we can wait for the grid to display the result and make it more stable, or fail it if the result is not displayed